src/aws_scanner_argument_parser.py

```diff
@@ -261,6 +261,7 @@ def _add_audit_route53_query_logs(self, subparsers: Any) -> None:
         self._add_accounts_args(audit_parser)
         self._add_verbosity_arg(audit_parser)
         self._add_enforce_arg(audit_parser, "add centralised query logs to Route53 Zones that don't already have one")
+        audit_parser.add_argument("-w", "--with_subscription_filter", type=bool, help="create subscription filter")
```

Comment:
this line looks common in 3

Might be time to extract to a helper method

Reply:
refactored accordingly